src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -54,7 +54,29 @@
 
 
 class FlowExecutor:
-    """This class is used to execute a single flow for different inputs."""
+    """This class is used to execute a single flow for different inputs.
+
+    :param flow: The flow to be executed.
+    :type flow: Flow
+    :param connections: The connections to be used for the flow.
+    :type connections: dict
+    :param run_tracker: The run tracker to be used for the flow.
+    :type run_tracker: RunTracker
+    :param cache_manager: The cache manager to be used for the flow.
+    :type cache_manager: AbstractCacheManager
```

Comment:
Same

Reply:
:type cache_manager: ~promptflow._core.cache_manager.AbstractCacheManager
There is _core for absolute import path.
![image](https://github.com/microsoft/promptflow/assets/17527303/346377c3-d2c8-455b-998e-af83e3852c50)